validation/promotion_test.go

```diff
@@ -10,15 +10,20 @@ import (
 func TestPromotion(t *testing.T) {
 	for _, chain := range OPChains {
 		chain := chain
+// WARNING: this test must not run along side any other tests, because it mutates some global object.
+// It should be strictly isolated. 
 		t.Run(perChainTestName(chain), func(t *testing.T) {
 			t.Parallel()
 			if chain.StandardChainCandidate {
+				// do not allow any test exclusions
+				exclusions = nil
```

Comment:
There might be a solution here where we lock a mutex on this object, and undo all the changes in this test before releasing the lock? @vdamle

Reply:
That is a great suggestion. I will give it a try!